trio/_core/_tests/test_io.py

```diff
@@ -305,12 +322,12 @@ async def test_wait_on_invalid_object():
             await wait(fileno)
 
 
-async def test_io_manager_statistics():
-    def check(*, expected_readers, expected_writers):
+async def test_io_manager_statistics() -> None:
+    def check(*, expected_readers: int, expected_writers: int) -> None:
         statistics = _core.current_statistics()
         print(statistics)
         iostats = statistics.io_statistics
-        if iostats.backend in ["epoll", "windows"]:
```

Comment:
Using a tuple here should work just as well

Reply:
Nvm, I forgot `in` doesn't narrow :(

Reply:
It could/should, just Mypy doesn't implement it yet.

Reply:
Yeah there's an issue about that :(